### PR TITLE
Basic updates to homepage design

### PR DESCRIPTION
**Pull request by jerryzhang222**
**Darkened the background image of the header:**

screen shot 2017-01-31 at 4 19 37 pm

**Changed team to have 4 team members on each row (sorry Min!), darkened secondary font, added a couple contributors to the bottom:**

screen shot 2017-02-01 at 1 57 18 pm

Other:

Changed Roboto Slab and Droid Serif (both serif fonts) to Droid Sans for a more unified, modern look.

### CODE REVIEW

File: _includes/team.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h2 class="section-heading">Our Amazing Team</h2>
             </div>
             <div class="row">
                 <div class="col-lg-8 col-lg-offset-2 text-center">
-                    <p class="large text-muted">Our experiences have inspired us to set out to build an intuitive, sophisticated tool to boost developer productivity. The Screwdriver team is based in Sunnyvale, CA.</p>
+                    <p class="large text-muted">Our experiences have inspired us to set out to build an intuitive, sophisticated tool to boost developer productivity. Other contributors include <a href="https://github.com/nicolaifsf">Nicolai Ferraris</a>, <a href="https://github.com/Filbird">Filbert Jahja</a>, and <a href="https://github.com/nkatzman">Noah Katzman</a>. The Screwdriver team is based in Sunnyvale, CA.</p>
```
Comment (d2lam) [inline _includes/team.html]: Should we make this a separate section with a list of names? I imagine this list might get longer and might not look too good in a paragraph.

Comment (tkyi) [inline _includes/team.html]: I think we can change it later if needed.

**Comment by y4h006969** [comment]
Love it.  Was actually thinking we should make a 3x3 grid.  Great follow up to #16.

File: _includes/team.html (hunk 1 above)
Comment (d2lam) [inline _includes/team.html]: Ik

**Comment by stjohnjohnson** [comment]
I really wish we didn't take up so much space with this.  I was hoping we could find a way to fit all the team members onto one or two rows.

**Comment by d2lam** [comment]
@stjohnjohnson  Does it? It takes up 3 rows for me.
@jerryzhang222  Can you update the screenshot?

**Comment by tkyi** [comment]
@d2lam I think John just wanted the pictures to not be as spaced out. We ended up agreeing on the 3 rows of 4 (with Min in her own row) for now.

Also I updated the screenshot.

**Comment by d2lam** [comment]
@tkyi  Ah okay

**Pull request merged**